apps/aecore/src/aec_sync.erl

```diff
@@ -106,7 +106,7 @@ compare_ping_objects(Local, Remote) ->
                           server_get_missing_blocks(Src);
                       _ ->
                           Dl = maps:get(<<"difficulty">>, Local),
-                          Dr = maps:get(<<"difficulty">>, Remote),
+                          Dr = maps:get(<<"difficulty">>, Remote, missing),
```

Comment:
This check shall be removed. Rather, the [`swagger.yaml` definition for `Ping`](https://github.com/aeternity/epoch/blob/bb5786751abdff67cc667092e7dd5e7ae45d6724/config/swagger.yaml#L360) shall be enriched with [`required`](https://github.com/OAI/OpenAPI-Specification/blob/1adcb8830aab48506b1ffd876330ab8c22820ad6/versions/2.0.md#schema-object).

Reply:
Will be addressed in [Pivotal story](https://www.pivotaltracker.com/story/show/153629617)

Reply:
The code below relies on inequality `>` of number vs. atom (`missing`) that may return unexpected result so I suggest changing at least this one in this PR.